pom.xml

```diff
@@ -781,6 +756,12 @@
                     <groupId>io.quarkus</groupId>
                     <artifactId>quarkus-extension-maven-plugin</artifactId>
                     <version>${quarkus.version}</version>
+                </plugin>                
+                <plugin>
+                    <groupId>org.apache.camel</groupId>
+                    <artifactId>camel-yaml-dsl-maven-plugin</artifactId>
+                    <!-- custom version needed for the restricted yaml dsl schema generation -->
+                    <version>3.20.3</version>
```

Comment:
Latest available productized version is 3.20.1

Reply:
The needed bits are only in 3.20.2, but I guess the productized bits include the required changes because of selected backports. We could double check

Reply:
@hdamarcelo is this the only missing version ?

Reply:
Yes, all the others that were changed have available productized versions. But since it was working before with a snapshot version I am testing building with this version to see if we run into problems.